x/evm/statedb/statedb.go

```diff
@@ -52,6 +53,45 @@ type StateDB struct {
 
 	// Per-transaction access list
 	accessList *accessList
+
+	// Transient storage
+	transientStorage transientStorage
+}
+
+func (s *StateDB) GetTransientState(addr common.Address, key common.Hash) common.Hash {
+	return s.transientStorage.Get(addr, key)
+}
+
+func (s *StateDB) SetTransientState(addr common.Address, key, value common.Hash) {
+	s.transientStorage.Set(addr, key, value)
+}
+
+func (s *StateDB) Prepare(rules params.Rules, sender, coinbase common.Address, dst *common.Address, precompiles []common.Address, list ethtypes.AccessList) {
```

Comment:
@Vvaradinov why is this needed?

Reply:
@fedekunze this were updates needed when bumping geth to our fork release branch.

Reply:
@fedekunze actually we can use the ` v1.10.26-evmos-rc1` release and those changes should not be needed